virtcontainers/kata_agent.go

```diff
@@ -1042,7 +1049,7 @@ func (k *kataAgent) handleEphemeralStorage(mounts []specs.Mount) []*grpc.Storage
 
 // handleBlockVolumes handles volumes that are block devices files
 // by passing the block devices as Storage to the agent.
-func (k *kataAgent) handleBlockVolumes(c *Container) []*grpc.Storage {
+func (k *kataAgent) handleBlockVolumes(sandbox *Sandbox, c *Container) []*grpc.Storage {
```

Comment:
Could you use the sandbox object embedded in the Container struct? `c.sandbox.config.AgentConfig...`, or are they different?

Reply:
That's a good idea, I've updated it.